doomrainbow/src/main/java/com/detroitlabs/kyleofori/doomrainbow/RangeLabelAlignment.java

```diff
@@ -0,0 +1,5 @@
+package com.detroitlabs.kyleofori.doomrainbow;
+
+public enum RangeLabelAlignment {
+    INWARD, OUTWARD, CENTERED
```

Comment:
these might be confusing if the user selected certain start/sweep angle combos, but i don't see a good way to work around it at the moment...